src/main/java/frc/robot/subsystems/arm/commands/SetArmsPosition.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.subsystems.arm.commands;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.arm.Arm;
+import         edu.wpi.first.math.MathUtil;
+import frc.robot.subsystems.arm.ArmConstants;
+
+public class SetArmsPosition extends CommandBase {
+    private final Arm arm;
+    private final Translation2d position;
+
+    public SetArmsPosition(Translation2d position) {
+        this.arm = Arm.getInstance();
+        this.position = position;
+        addRequirements(arm);
+    }
+
+    @Override
+    public void initialize() {
+        arm.setEndPosition(position);
+    }
+
+    @Override
+    public boolean isFinished() {
+       return MathUtil.applyDeadband(position.getX(), ArmConstants.SETPOINT_DEADBAND) == arm.getEndPosition().getX() && MathUtil.applyDeadband(position.getY(), ArmConstants.SETPOINT_DEADBAND) == arm.getEndPosition().getY();
```

Comment:
I'm really sorry for misleading you yesterday, the way you check that they have reached the setpoint is by checking that the difference is in the deadband `applyDeadband(Ax-Bx) == 0` (pseudocode). It applies to the other commands as well